betfair/models.py

```diff
@@ -138,7 +138,7 @@ class Match(BetfairModel):
 
 
 class Runner(BetfairModel):
-    selection_id = Field(DataType(float), required=True)
+    selection_id = Field(DataType(six.text_type), required=True)
```

Comment:
the patch for selection_id is not important. AFAIR I just took the change from someone else and changed in all occurences for consistency. 
Note that there are also some other fixes in this patch however.